Sources/PenguinStructures/Graphs/AdjacencyList.swift

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/// AdjacencyList is a general-purpose, basic graph implementation.
+/// A general-purpose, flexible [adjacency list](https://en.wikipedia.org/wiki/Adjacency_list).
```

Comment:
So I guess it's not all that flexible if it's just arrays and it can't store properties, and I suppose "general purpose" isn't adding much, and I guess I would want "graph" in the description somewhere just to give the non-link-hopping reader some context. How about:
```suggestion
/// An [adjacency list](https://en.wikipedia.org/wiki/Adjacency_list) graph.
```

More accurate/descriptive, but I don't love it because it suggests it's not-a graph:

```suggestion
/// The vertex/edge structure of an arbitrary [adjacency list](https://en.wikipedia.org/wiki/Adjacency_list) graph.
```